File: source/server/types/Dist.go
package types

// Dist 字典表
type Dist struct {
	Id        int64  `json:"id"`
	Type      string `json:"type"`
	DistKey   string `json:"distKey"`
	DistValue string `json:"distValue"`
	Sort      int64  `json:"sort"`
	BookKey   string `json:"bookKey"`
}

type DistQuery struct {
	Id        int64  `json:"id" form:"id"`
	Type      string `json:"type" form:"type"`
	BookKey   string `json:"bookKey"`
	DistKey   string `json:"distKey" form:"distKey"`
	DistValue string `json:"distValue" form:"distValue"`
	PageNum   int64  `json:"pageNum" uri:"pageNum" form:"pageNum"`
	PageSize  int64  `json:"pageSize" uri:"pageSize" form:"pageSize"`
}

File: source/books/src/types/model/dist.ts
export interface Dist {
  id?: number;
  type?: string;
  distKey?: string;
  distValue?: string;
  bookKey?: string;
  sort?: number;
}

/**
 * 添加字典
 */
export interface CreateDistDto {
  type?: string;
  distKey?: string;
  distValue?: string;
  sort?: number;
}


/**
* 修改字典
*/
export interface UpdateDistDto {
  type?: string;
  distKey?: string;
  distValue?: string;
  sort?: number;
}
export class DistQuery {
  pageNum: number = 1;
  pageSize: number = 10;
  id?: number;
  type?: string;
  distKey?: string;
  distValue?: string;
}
